src/features/user/userSlice.ts

```diff
@@ -0,0 +1,37 @@
+﻿import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
```

Comment:
**nitpick:** Let's structure this `data` layer so we know it's all related things are in one place. So:

`src/data/user/dto.ts`
`src/data/user/slice.ts`

Reply:
@Sumick done :)